### PR TITLE
fix/DE21934-21935-21936-21937-219348: Fixed the Prod Reported issue's

### PR DESCRIPTION
**Pull request by roshan-bisht-lab49**

### CODE REVIEW

**Review by damiano-melcarne-lab49 — CHANGES_REQUESTED** [review]
For `DE21937 `- The Minimum Proposal Deposit field should be displayed under the "Voting" stepper in the Create a DAO wizard. This field can be read-only if the amount is fixed in the smart contracts.

Screen Shot 2023-07-25 at 2 41 50 PM

File: src/dex-ui/routes/router.tsx [hunk 1]
```diff
@@ -119,6 +119,14 @@ export const router = createBrowserRouter(
           <Route path={Paths.DAOs.Members} element={<MembersList />} />
           <Route path={Paths.DAOs.Settings} element={<DAOSettings />} />
         </Route>
+        <Route
```
Comment (damiano-melcarne-lab49) [inline src/dex-ui/routes/router.tsx:122]: 1) I believe we want the DAO Details Proposal to show in the list of proposal types.
Screen Shot 2023-07-25 at 3 01 45 PM

2) The "Change" button should navigate the user to the second step in the new proposal wizard with the type preselected.

**Review by damiano-melcarne-lab49 — CHANGES_REQUESTED** [review]

**Review by damiano-melcarne-lab49 — CHANGES_REQUESTED** [review]
This was not originally listed as a requirement in `DE21934` but I'm adding this here since it's closely related.

The Initial Token Supply shown in the Create A DAO review form should show the user-entered amount (rounded by the decimal amount):

**Review by damiano-melcarne-lab49 — CHANGES_REQUESTED** [review]
This was not originally listed as a requirement in `DE21934` but I'm adding this here since it's closely related.

The Initial Token Supply shown in the Create A DAO review form should show the user-entered amount (rounded by the decimal amount):
Screen Shot 2023-07-25 at 2 32 19 PM

File: src/dex-ui/routes/router.tsx (hunk 1 above)
Comment (damiano-melcarne-lab49) [reply src/dex-ui/routes/router.tsx:122]: _A note about point 2 above:_ I left a note in Figma for Tunde to determine if we should hide the "Type" step in the Create Proposal wizard when a user navigates to the wizard using a shortcut button. For now, we should follow the same pattern we're using for other proposal wizard shortcut buttons. If needed, we can hide the "Type" step in another task.

Comment (roshan-bisht-lab49) [reply src/dex-ui/routes/router.tsx:122]: @damiano-melcarne-lab49 I have addressed all feedbacks just for above one I am planning to do it in separate PR, if that is fine.?

cc: @amandeeprehal-lab49

**Review by amandeeprehal-lab49 — APPROVED** [review]

**Pull request merged**